src/Symfony/Component/HttpKernel/DataCollector/RequestDataCollector.php

```diff
@@ -40,12 +40,9 @@ public function __construct()
     public function collect(Request $request, Response $response, \Exception $exception = null)
     {
         $responseHeaders = $response->headers->all();
-        $cookies = array();
+        $responseCookies = array();
         foreach ($response->headers->getCookies() as $cookie) {
-            $cookies[] = $this->getCookieHeader($cookie->getName(), $cookie->getValue(), $cookie->getExpiresTime(), $cookie->getPath(), $cookie->getDomain(), $cookie->isSecure(), $cookie->isHttpOnly());
-        }
-        if (count($cookies) > 0) {
-            $responseHeaders['Set-Cookie'] = $cookies;
```

Comment:
@fabpot now #20569 is merged, we should not forget to update this.. i kinda did ;-) and it will cause duplicate headers i guess.